packages/components/src/components/route-instructions-step/route-instructions-step.tsx

```diff
@@ -55,6 +55,12 @@ export class RouteInstructionsStep implements ComponentInterface {
      */
     @Prop() fromTransitStop: string;
 
+    /**
+     * The destination location.
```

Comment:
Should we put a few words in what this is used for?

Reply:
✅